.husky/post-checkout

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+# Checks branch name for Jira ticket id
+packages/tools/pie-monorepo-utils/git-hooks/check-branch-name.js $(git symbolic-ref --short HEAD)
```

Comment:
Utilising the `post-checkout` hook here so invalid branch names are caught earlier in the dev process.